### PR TITLE
TRUNK-5733: Add tests for Location class

### PR DESCRIPTION
**Pull request by Derplicity**
## Description of what I changed
Added tests to LocationTest.java that cover hasTag() and getChildLocations() methods.

## Issue I worked on
see https://issues.openmrs.org/browse/TRUNK-5733

## Checklist: I completed these to help reviewers :)

- [x] My pull request only contains **ONE single commit**

- [x] My IDE is configured to follow the [**code style**]

- [x] I have **added tests** to cover my changes. (If you refactored
  existing code that was well tested you do not have to add tests)

- [x] I ran `mvn clean package` right before creating this pull request and
  added all formatting changes to my commit.

- [x] All new and existing **tests passed**.

- [x] My pull request is **based on the latest changes** of the master branch.

### CODE REVIEW

File: api/src/test/java/org/openmrs/LocationTest.java [hunk 1]
```diff
@@ -102,4 +103,89 @@ public void getDescendantLocations_shouldReturnNonRetiredDescendantLocationsIfIn
...
+		
+		LocationTag tagOne = new LocationTag("One", "");
+		LocationTag tagTwo = new LocationTag("Two", "");
+		
+		location.addTag(tagOne);
+		location.addTag(tagTwo);
+		
+		String tagToFind = "Two";
+		
+		assertTrue(location.hasTag(tagToFind));
```
[the remark this answers is not in the document]
Comment (Derplicity) [reply api/src/test/java/org/openmrs/LocationTest.java:124]: Hey @jnsereko
Should I get rid of all gaps?

[the remark this answers is not in the document]
Comment (jnsereko) [reply api/src/test/java/org/openmrs/LocationTest.java:124]: Not all gaps, but there are some unnecessary gaps.
May you please perform some  [code reviews](https://wiki.openmrs.org/display/docs/Code+Review) (viewing other people`s code) or take a look at some  **openmrs-core-API  test classes**.  I am sure u will get a view of which gaps to retain

**Comment by HerbertYiga** [comment]
Add a ticket Id on your commit as recommended here https://wiki.openmrs.org/display/docs/Pull+Request+Tips

File: api/src/test/java/org/openmrs/LocationTest.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (Derplicity) [reply api/src/test/java/org/openmrs/LocationTest.java:123]: @jnsereko I removed the gaps that I found unneeded.

**Comment by jnsereko** [comment]
I have just realized that the Ticket You are working on is in a **Needs Assessment** state
i have left a comment for you on the ticket

**Comment by Derplicity** [comment]
> did you implement advice from **Herbert**

@jnsereko Yes, I changed the commit message to include the ticket id.

**Pull request closed without merging**